### PR TITLE
added view-comment component and integrated api

### PR DESCRIPTION
**Pull request by etamity**
https://trello.com/c/e6EnL8I0/807-3-show-comments-add-comment-module-for-new-world

### CODE REVIEW

File: src/elements/kw-social-comment/kw-social-comment.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            .orangebutton {
+                background-color: var(--color-orange);
+                border-radius: 5px;
+                width: 79px;
+                height: 32px;
+                color: var(--color-white);
+                border: none;
+            }
+            
+            .commentbutton {
```
Comment (paulvarache) [inline src/elements/kw-social-comment/kw-social-comment.html]: No casing?

File: src/elements/kw-social-comment/kw-social-comment.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                width: 100px;
+            }
+            
+            .commentbutton img {
+                position: relative;
+                top: 2px;
+                left: 2px;
+            }
+            
+            .commentsBox {
```
Comment (paulvarache) [inline src/elements/kw-social-comment/kw-social-comment.html]: Camel case?

File: src/elements/kw-social-comment/kw-social-comment.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                color: var(--color-grey-dark);
+            }
+            
+            .commentsBox p {
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+                font-size: 16px;
+            }
+            
+            iron-scroll-threshold {
```
Comment (paulvarache) [inline src/elements/kw-social-comment/kw-social-comment.html:128]: No need here, it is never used in the template

File: src/elements/kw-social-comment/kw-social-comment.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+                share: {
+                    type: Object
+                }
+            },
+            attached () {
+                this.page = 0;
+                this.loadData();
+            },
+            loadData(){
+                // When we hit the last page
```
Comment (paulvarache) [inline src/elements/kw-social-comment/kw-social-comment.html:176]: There no page here. This is code left from the copy and paste from the feed. Please clean it

File: src/elements/kw-social-comment/kw-social-comment.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+                if (!Boolean(this.share)) {
+                    return;
+                }
+                fetch(this._getUrl('comments', { type: this.share.type ,id: this.share.id}))
+                .then(r => r.json())
+                .then(r => {
+                    r.entries.forEach(comment => {
```
Comment (paulvarache) [inline src/elements/kw-social-comment/kw-social-comment.html:183]: Same thing here. No need to add elements one by one if not in the context of an infinite scroll.

File: src/elements/kw-share-detail/demo/index.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
+<html>
```
Comment (paulvarache) [inline src/elements/kw-share-detail/demo/index.html:1]: `kano-share-detail` is still code that belongs to another PR

File: src/elements/kw-share-detail/kw-share-detail.html [hunk 7]
```diff
@@ -1,101 +1,215 @@
...
-                        type: Object
-                    }
-                },
-                observers: [
-                    '_shareAppChanged(share.app)'
-                ],
-                getShareElement () {
-                    return this;
-                },
-                _shareAppChanged (shareApp) {
```
Comment (paulvarache) [inline src/elements/kw-share-detail/kw-share-detail.html:67]: Where did that code go? It is important to load the right share component

**Comment by etamity** [comment]
Hi Paul, due to this two tickets are relative, I have to merge it to adjust the code, this is the latest update
